src/services/walletConnect.ts

```diff
@@ -423,16 +462,16 @@ export const useWalletConnect = () => {
       if (request.method === ETH_SEND_TX) {
         const sendTransactionToBatch = async () => {
           try {
-            addToBatch({
+            showTransactionConfirmation({
               title: 'WalletConnect transaction',
-              description: '',
-              chainId: chainIdNumber,
-              to: checksumAddress(request.params[0].to),
-              value: formatEther(hexToBigInt(request.params[0].value)),
-              data: request.params[0].data,
+              description: 'This will execute a transaction coming from a dApp',
```

Comment:
we'll need to change this message, can you include the dapp name here that is requested the transaction please? 

```suggestion
              description: '{dappName} wants to send a transaction',
```